src/server/utilities/liteATITracking/clickTracking/index.ts

```diff
@@ -1,4 +1,6 @@
 export default () => {
+  const clicktrackingFiredUrls: string[] = [];
```

Comment:
Could we rename this for consistency with `viewTracking`?
```suggestion
  const firedUrls: string[] = [];
```

Reply:
Hello, if we have both scripts on the same page, do we risk these two arrays conflicting with each other, since clicktrackingFiredUrls may effectively be in the same scope?

Reply:
Ohhhh interesting. I think not since they are each within the scope of different functions, but I am OK to leave this as is, just in case it's not.